airbyte-webapp/src/pages/SourcesPage/pages/CreateSourcePage/components/SourceForm.tsx

```diff
@@ -53,19 +60,28 @@ export const SourceForm: React.FC<SourceFormProps> = ({ onSubmit, sourceDefiniti
     });
   };
 
+  const showBuilderNavigationLinks = useExperiment("connectorBuilder.showNavigationLinks", false);
+
   return (
-    <ConnectorCard
-      formType="source"
-      title={<FormattedMessage id="onboarding.sourceSetUp" />}
-      description={<FormattedMessage id="sources.description" />}
-      isLoading={isLoading}
-      fetchingConnectorError={sourceDefinitionError instanceof Error ? sourceDefinitionError : null}
-      availableConnectorDefinitions={sourceDefinitions}
-      onConnectorDefinitionSelect={onDropDownSelect}
-      selectedConnectorDefinitionSpecification={sourceDefinitionSpecification}
-      selectedConnectorDefinitionId={sourceDefinitionId}
-      onSubmit={onSubmitForm}
-      jobInfo={LogsRequestError.extractJobInfo(error)}
-    />
+    <>
+      <ConnectorCard
+        formType="source"
+        title={<FormattedMessage id="onboarding.sourceSetUp" />}
+        description={<FormattedMessage id="sources.description" />}
+        isLoading={isLoading}
+        fetchingConnectorError={sourceDefinitionError instanceof Error ? sourceDefinitionError : null}
+        availableConnectorDefinitions={sourceDefinitions}
+        onConnectorDefinitionSelect={onDropDownSelect}
+        selectedConnectorDefinitionSpecification={sourceDefinitionSpecification}
+        selectedConnectorDefinitionId={sourceDefinitionId}
+        onSubmit={onSubmitForm}
+        jobInfo={LogsRequestError.extractJobInfo(error)}
+      />
+      {showBuilderNavigationLinks && !sourceDefinitionSpecification && (
```

Comment:
Only shows the BuilderPrompt if the experiment is enabled, and if there is no source type selected